internal/controller/vrg_volrep.go

```diff
@@ -1806,6 +1771,26 @@ func setPVCClusterDataProtectedCondition(protectedPVC *ramendrv1alpha1.Protected
 	}
 }
 
+func (v *VRGInstance) updatePVCLastSyncCounters(pvcNamespace, pvcName string, status *volrep.VolumeReplicationStatus) {
+	protectedPVC := v.findProtectedPVC(pvcNamespace, pvcName)
+	if protectedPVC == nil {
+		return
+	}
+
+	if status == nil {
+		protectedPVC.LastSyncTime = nil
+		protectedPVC.LastSyncDuration = nil
+		protectedPVC.LastSyncBytes = nil
```

Comment:
do you need to explicitly set these values if status is nil. Isn't the default value nil.

Reply:
Keep the current behavior in this pr. Optimizing unneeded assignments is not in the scope of this change.

Reply:
@rakeshgm , we can add more optimizations later. Let's keep the assignments behaviour for now

Reply:
ok, LGTM then.